pkg/node/node.go

```diff
@@ -236,6 +236,17 @@ func NewBee(interrupt chan os.Signal, addr string, publicKey *ecdsa.PublicKey, s
 		return nil, err
 	}
 
+	var unreserveFn func([]byte, uint8) (uint64, error)
+	var evictFn = func(b []byte) error {
+		_, err := unreserveFn(b, swarm.MaxPO+1)
+		return err
+	}
+
+	batchStore, err := batchstore.New(stateStore, evictFn, logger)
```

Comment:
This might cause some problems. If the node is started in a mode where there is no chain backend then we end up with this component dangling. It would be good to make sure that the component can finally shut down. Possibly also to initialize a mock of the component when the node needs to start without a chain backend

Reply:
replaced for a `noOp` implementation when in chainless mode, not sure what to do about the shutdown part...